bin/metrics-pdns.rb

```diff
@@ -37,15 +37,51 @@ class PdnsGraphite < Sensu::Plugin::Metric::CLI::Graphite
          long: '--scheme SCHEME',
          default: "#{Socket.gethostname}.pdns"
 
-  def run
-    metrics = `sudo rec_control get-all`.split("\n")
+  option :syslog_path,
+         description: 'Path to the syslog',
+         long: '--syslog SYSLOG',
+         default: '/var/log/messages'
+
+  option :extra_stats,
+         description: 'flag to send stats collected from syslog like qps',
+         short: '-e EXTRA',
+         long: '--extra EXTRA',
+         default: false
```

Comment:
can you add `boolean: true` and take off the `EXTRA` part of the short and log args since it's a boolean and doesn't need it.

Reply:
sure